mpf/devices/score_reel.py

```diff
@@ -154,7 +157,8 @@ async def _advance_reel_if_position_does_not_match(self):
         while self._destination_value != self.assumed_value:
             self.machine.events.post('reel_{}_will_advance'.format(self.name))
             wait_ms = self.config['coil_inc'].pulse(max_wait_ms=500)
-            self.machine.events.post('reel_{}_advancing'.format(self.name))
+            if not self._quiet:
+                self.machine.events.post('reel_{}_advancing'.format(self.name))
```

Comment:
This is a nice fix, but limited to your specific application. An alternative approach would be to post the event with an argument like `reset: true` to indicate whether the reel is resetting or not. This way there's still an event for people who want/need to know the advancement of the reel, and chimes can be made conditional on that argument to determine whether to chime or not.